src/Tests/QueryExecutor.cs

```diff
@@ -7,6 +9,8 @@ static class QueryExecutor
 {
     public static async Task<string> ExecuteQuery(string queryString, Inputs? inputs, ValidatorTypeCache cache)
     {
+        Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-US");
```

Comment:
This was required to stabilize the tests, since the error messages depend on the user locale.